lib/exposes.js

```diff
@@ -303,12 +303,14 @@ module.exports = {
     presets: {
         action: (values) => new Enum('action', access.STATE, values).withDescription('Triggered action (e.g. a button click)'),
         aqi: () => new Numeric('aqi', access.STATE).withDescription('Air quality index'),
+        auto_off: () => new Binary('auto_off', access.STATE, true, false).withDescription('Indicates whether the device has auto off with low consumer load functionality enabled or not'),
```

Comment:
access should be `access.STATE_SET` here (since you can set it via toZigbee), same goes for `power_outage_memory` and `led_disabled`.

Reply:
Missed it, will fix

Reply:
Correct me, if I'm wrong, but I understand this way:
STATE_SET - only for set
STATE_GET - only for get
STATE - for both

Reply:
And I have more strategical question:
Should be such states not binary, but like a switch - literally "on/off' ?